opa/models/session.py

```diff
@@ -0,0 +1,60 @@
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class Session(models.Model):
+    _name = 'opa.session'
+    _description = 'Session from an Open Academy course'
+
+    name = fields.Char(string='Title', required=True)
+    start_date = fields.Date(string='Started on', default=fields.Date.today, required=True)
+    end_date = fields.Date(string='Ended on', default=fields.Date.today)
```

Comment:
similar case

Reply:
From this point on I will no longer mention this problem, fix it for the rest of the cases.